### PR TITLE
rose_arch: STDERR for target with bad status

### PR DESCRIPTION
**Pull request by matthewrmshin**
This should make it more obvious to users when we have a target with no source from its list of optional sources and/or an optional target with no source.

@benfitzpatrick @kaday please review.

### CODE REVIEW

File: lib/python/rose/apps/rose_arch.py [hunk 1]
```diff
@@ -295,7 +295,10 @@ def _run_target_update(cls, dao, app_runner, compress_manager, target):
         if target.status in (target.ST_BAD, target.ST_NULL):
             # boolean to int
             target.command_rc = int(target.status == target.ST_BAD)
-            app_runner.handle_event(RoseArchEvent(target))
+            level = int(target.status != target.ST_BAD)
```
Comment (benfitzpatrick) [inline lib/python/rose/apps/rose_arch.py]: This is neat but not obvious! Could we have something more explicit like:
```python
if target.status == target.ST_BAD:
    level = Event.FAIL
else:
    level = Event.DEFAULT
```

Comment (matthewrmshin) [inline lib/python/rose/apps/rose_arch.py]: Done.

**Comment by benfitzpatrick** [comment]
Looks fine.

**Comment by kaday** [comment]
Looks good to me.
Passes test battery in my environment.

**Pull request merged**